### PR TITLE
feat: new filenames structure

### PR DESCRIPTION
**Pull request by clarkdo**
- [x] Update document

### CODE REVIEW

File: lib/common/nuxt.config.js [hunk 1]
```diff
@@ -51,12 +51,13 @@ export default {
...
     publicPath: '/_nuxt/',
     filenames: {
-      app: '[name].[contenthash].js',
-      chunk: '[name].[contenthash].js',
-      css: '[name].[contenthash].css',
-      img: 'img/[name].[hash:7].[ext]',
-      font: 'fonts/[name].[hash:7].[ext]',
-      video: 'videos/[name].[hash:7].[ext]'
+      // { isDev, file, isClient, isServer }
+      app: ({ isDev }) => isDev ? '[path][name].js' : '[contenthash].js',
```
Comment (pi0) [inline lib/common/nuxt.config.js]: Can we use `chunkhash`?

File: lib/common/nuxt.config.js [hunk 2]
```diff
@@ -51,12 +51,13 @@ export default {
...
     publicPath: '/_nuxt/',
     filenames: {
-      app: '[name].[contenthash].js',
-      chunk: '[name].[contenthash].js',
-      css: '[name].[contenthash].css',
-      img: 'img/[name].[hash:7].[ext]',
-      font: 'fonts/[name].[hash:7].[ext]',
-      video: 'videos/[name].[hash:7].[ext]'
+      // { isDev, isClient, isServer }
+      app: ({ isDev }) => isDev ? '[path][name].js' : '[contenthash].js',
```
Comment (pi0) [inline lib/common/nuxt.config.js]: Can we use `chunkhash` for app and chunk?

**Review by pi0 — APPROVED** [review]

**Comment by manniL** [comment]
@clarkdo Is it included in the 2.0 docs yet? :thinking:

**Comment by clarkdo** [comment]
@manniL https://github.com/nuxt/docs/commit/1eca53bf7974cd74115485be4ada2f7ab59e2f22

**Comment by manniL** [comment]
@clarkdo Thank you :relaxed:

**Pull request merged**